contracts/Tavern.sol

```diff
@@ -64,19 +64,20 @@ contract Tavern is ITavern, Pausable {
         uint32 _seekerId,
         uint32 _solverId,
         uint256 _paymentAmount,
-        string memory infoURI
+        string memory infoURI,
+        uint256 _maxExtensions
     ) external payable onlyBarkeeper {
         IQuest quest = IQuest(Clones.clone(questImplementation));
         address escrowImpl = escrowNativeImplementation;
         address taxManager = INexus(nexus).taxManager();
 
         require(taxManager != address(0), "TaxManager not set");
-   
+
         emit QuestCreatedNative(
-            _seekerId, 
-            _solverId, 
-            address(quest), 
-            escrowImpl, 
+            _seekerId,
+            _solverId,
+            address(quest),
+            escrowImpl,
```

Comment:
should we add max extensions count to this event?

Reply:
Added maxExtension to the event